Lib/glyphsLib/parser.py

```diff
@@ -101,17 +101,25 @@ def _parse_dict_into_object(self, res, d):
 
 
 def load_glyphspackage(package_dir):
-    infofile = os.path.join(package_dir, "fontinfo.plist")
-    orderfile = os.path.join(package_dir, "order.plist")
+    package = Path(package_dir)
+    infofile = package / "fontinfo.plist"
+    orderfile = package / "order.plist"
```

Comment:
if in the end the order.plist is (unlike the UFO contents.plist) just way to sort glyphs but not an actual index that fully determines what's in the font, can it be omitted altogether?

Reply:
Keeping the glyphs in their expected order is important though, isn't it?

Reply:
well, not as important as keeping it at all. Just curious, does it actually load in Glyphs.app if you omit one?

Reply:
and then, there also the `glyphOrder` custom parameter that one can set on the GSFont that basically accomplishes the same thing..

Reply:
but I think our current code expects the order.plist to be present...
we unconditionally do:

```
glyphorder = openstep_plist.load(open(orderfile, "r", encoding="utf-8"))
```

Reply:
by the way, it's good practice when opening a file to close it afterwards, here and below we just open. Ideally we do `with open(...):`

Reply:
you said

> If there is no order.plist, then Glyphs.app will still load the file

so I'd expect that we match Glyphs.app behavior, whatever their spec says